tgui/packages/tgui/links.js

```diff
@@ -13,17 +13,17 @@ export const captureExternalLinks = () => {
     /** @type {HTMLElement} */
     let target = e.target;
     // Recurse down the tree to find a valid link
-    while (target && target !== document.body) {
+    while (true) {
+      // Reached the end, bail.
+      if (!target || target === document.body) {
+        return;
+      }
```

Comment:
I don't quite understand why you can't move this if check into the conditional of the while loop

Reply:
It is necessary to bail based on this condition, not break the loop.